### PR TITLE
EMBD-632 Unify Bootloader Scripts

### PR DESCRIPTION
**Pull request by teginyucelmayadagli**
# Description
Bootloader python script returns error if any

# Changes
- Added return status for called functions

# Test Plan
Can be tested on any ActPack or Exo with bootloader. 
https://dephy.com/wiki/flexsea/doku.php?id=actpackfieldupdate

# Expected Results
Shall return bootloader status 

# Requirements
- Flexsea-build (Same branch name EMBD-632)

### CODE REVIEW

File: Python/flexsea_demo/bootloader.py [hunk 1]
```diff
@@ -65,6 +65,8 @@ def main():
 	# pylint: disable=import-outside-toplevel
 	import argparse
 
+	result = False
```
Comment (AliaLayne) [inline Python/flexsea_demo/bootloader.py:68]: If the line:

result = bootloader(flex.FlexSEA(), args.port[0], args.baud_rate, args.target)

is sending back a value.  Won't the value in the line above be overwritten?  If so, do you need it?  If you do need it, maybe there is something wrong with the bootloader function?

Just an observation.

**Review by AliaLayne — APPROVED** [review]
I don't know if this was done already! But here's my 2 cents.  In addition to my single comment.  I would consider adding some more comments throughout the code.  It doesn't need many.  The variable names are fully written out, which makes it very understandable without them.  But I do think a few would be helpful.

File: Python/flexsea_demo/bootloader.py (hunk 1 above)
Comment (JFDuval) [reply Python/flexsea_demo/bootloader.py:68]: @teginyucelmayadagli can you take a look at this? Fix or resolve

**Review by JFDuval — APPROVED** [review]
Looks good to me, and works. Please address Alia's comment before merging.

File: Python/flexsea_demo/bootloader.py (hunk 1 above)
Comment (Sotilrac) [reply Python/flexsea_demo/bootloader.py:68]: You can loose the result variable altogether, just `return bootloader(flex.FlexSEA(), args.port[0], args.baud_rate, args.target)` at the end

**Review by Sotilrac — CHANGES_REQUESTED** [review]

File: Python/flexsea_demo/bootloader.py (hunk 1 above)
Comment (teginyucelmayadagli) [reply Python/flexsea_demo/bootloader.py]: Addressed the issue

**Review by Sotilrac — APPROVED** [review]

**Pull request merged**